File: backend/backend-main/models/Response.js
const mongoose = require('mongoose');


const Response = mongoose.model('Response',{
    content: {
        type:String,
        require : true
    },
    id_admin: {
        type:integer,
        require : true
    },
    id_rec: {
        type: integer,
        // require : true
    }
    
})

module.exports = Reclamation;

File: Ecocitoyen_frontend/app/src/main/java/com/example/ecocitoyen_frontend/Models/Response.kt
package com.example.ecocitoyen_frontend.Models

class Response(var id: Int,
               var content:String,
               var id_admin:Int,
               var id_rec:Int) {
    override fun toString(): String {
        return "Response(id=$id, content='$content', id_admin=$id_admin, id_rec=$id_rec)"
    }
}